modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -24,6 +24,11 @@ export interface ExecCommandArgs {
   skip_focus?: boolean;
 }
 
+// List of commands that are considered safe even if the editor has no selection when the iframe is hidden in Firefox
+const selectionSafeCommands = [ 'toggleview' ];
```

Comment:
Can we put information in the JIRA (and maybe tag this line with the JIRA) of exactly the issue that was encountered when this wasn't here? I assume it was just something like the command didn't work on Firefox because the iframe was hidden by the code view and therefore it was blocked? Was it Firefox specific? 

Apologies if you have already written this somewhere.

Reply:
Added details to the JIRA and a reference to the comment. Not sure I can make that comment more clear though without writing an essay on something that should be rather clear from just following the code.

Reply:
Very happy with that JIRA detail. Thanks.